pylint/lint/run.py

```diff
@@ -65,6 +65,18 @@ def _query_cpu() -> int | None:
             cpu_shares = int(file.read().rstrip())
         # For AWS, gives correct value * 1024.
         avail_cpu = int(cpu_shares / 1024)
+    elif Path("/sys/fs/cgroup/cpu.max").is_file():
```

Comment:
Shouldn't this take precedence over `cpu.shares`? If the new file is present it should probably be preferred over the old one?

Reply:
I wouldn't expect them to co-exist so just went with a quick grouping around v1 vs v2 but it probably makes sense just in case. 
I've pushed a new commit which prefers the v2 files and I think also makes the logic a bit clearer